JetNews/app/src/main/java/com/example/jetnews/ui/home/HomeRoute.kt

```diff
@@ -121,71 +106,60 @@ fun HomeRoute(
         }
     }
 
-    BoxWithConstraints {
-        // Determine which type of the home screen to display
-        val windowSize = remember(maxWidth) { getWindowSize(maxWidth) }
-        val homeScreenType = getHomeScreenType(windowSize, uiState)
-        // Modifier to be applied to the all HomeScreen types
-        val screenModifier = Modifier.homeScreenPadding(showNavRail)
-
-        when (homeScreenType) {
-            HomeScreenType.FeedWithArticleDetails -> {
-                HomeFeedWithArticleDetailsScreen(
-                    uiState = uiState,
-                    showNavRail = showNavRail,
-                    onToggleFavorite = onToggleFavorite,
-                    onSelectPost = onSelectPost,
-                    onRefreshPosts = onRefreshPosts,
-                    onErrorDismiss = onErrorDismiss,
-                    onInteractWithList = onInteractWithFeed,
-                    onInteractWithDetail = onInteractWithArticleDetails,
-                    openDrawer = openDrawer,
-                    navigateToInterests = navigateToInterests,
-                    homeListLazyListState = homeListLazyListState,
-                    articleDetailLazyListStates = articleDetailLazyListStates,
-                    scaffoldState = scaffoldState,
-                    modifier = screenModifier
-                )
-            }
-            HomeScreenType.Feed -> {
-                HomeFeedScreen(
-                    uiState = uiState,
-                    showNavRail = showNavRail,
-                    onToggleFavorite = onToggleFavorite,
-                    onSelectPost = onSelectPost,
-                    onRefreshPosts = onRefreshPosts,
-                    onErrorDismiss = onErrorDismiss,
-                    openDrawer = openDrawer,
-                    navigateToInterests = navigateToInterests,
-                    homeListLazyListState = homeListLazyListState,
-                    scaffoldState = scaffoldState,
-                    modifier = screenModifier
-                )
-            }
-            HomeScreenType.ArticleDetails -> {
-                // Guaranteed by above condition for home screen type
-                check(uiState is HomeUiState.HasPosts)
+    // Modifier to be applied to the all HomeScreen types
```

Comment:
I don't understand this comment. What modifier?

Reply:
This comment shouldn't be there. Removed, thanks!